_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -175,4 +172,5 @@ This `title` element is a child of an `svg` element.
 </svg>
 ```
 
-[html]: #namespaced-element
+[document title]: https://html.spec.whatwg.org/multipage/dom.html#document.title 'HTML definition of Document Title'
```

Comment:
This should rather point to
```suggestion
[document title]: https://html.spec.whatwg.org/multipage/dom.html#the-title-element-2 'HTML definition of Document Title'
```
(i.e. "the first `title` element …")

And we might want to accordingly change or remove the AS "assumption" that browsers do use it correctly if we take a definition from the specs.